package.json

```diff
@@ -1322,7 +1348,7 @@
         },
         {
           "command": "confluent.schemas.refresh",
-          "when": "view == confluent-schemas",
+          "when": "view == confluent-schemas && confluent.schemaRegistrySelected",
           "group": "navigation@5"
         },
```

Comment:
Also unrelated cleanup, not sure why we would refresh a view that had no resource selected.